jdaviz/core/helpers.py

```diff
@@ -265,3 +267,106 @@ def get_model_parameters(self, models=None, model_label=None, x=None, y=None):
                     param_units[key].get(param_name, None))
 
         return parameters_cube
+
+    def show_inline(self):
+        """
+        Display the jdaviz application inline in a notebook.  Note this is
+        functionally equivalent to displaying the cell ``self.app`` in the
+        notebook.
+
+        See Also
+        --------
+        show_in_sidecar
+        show_in_new_tab
+        """
+        display(self.app)
+
+    def show_in_sidecar(self, **kwargs):
+        """
+        Display the jdaviz application in a "sidecar", which by default is a tab
+        on the right side of the jupyterlab  interface.
+
+        Parameters
+        ----------
+        title : str, optional
+            The title of the sidecar tab.  Defaults to the name of the
+            application - e.g. "specviz".
+        anchor : str
+            Where the tab should appear, by default on the right.  To see all
+            options see the ``sidecar.Sidecar.anchor.values``.
+
+        additional keywords are passed into the ``sidecar.Sidecar`` constructor.
+        See ``jupyterlab-sidecar`` for the most up-to-date options.
+
+        Returns
+        -------
+        sidecar
+            The ``sidecar.Sidecar`` object used to create the tab.
+
+        Notes
+        -----
+        The ``jupyterlab-sidecar`` module needs to be installed for this method
+        to work.  ``pip install sidecar`` should suffice for most use cases.
+
+        If this method is called in the "classic" Jupyter notebook, the app will
+        appear inline, as only lab has a mechanism to have multiple tabs.
+
```

Comment:
Unnecessary. You already made it a required dependency in `setup.cfg`.

```suggestion
        Notes
        -----
        If this method is called in the "classic" Jupyter notebook, the app will
        appear inline, as only lab has a mechanism to have multiple tabs.
```